terraform/groups/ecs-service/locals.tf

```diff
@@ -51,7 +51,6 @@ locals {
     { "name": "COOKIE_NAME", "value": "${var.cookie_name}" },
     { "name": "LOG_LEVEL", "value": "${var.log_level}" },
     { "name": "NODE_ENV", "value": "${var.node_env}" },
-    { "name": "RESTRICTED_WORD_WEB_PORT", "value": "${var.restricted_word_web_port}" },
```

Comment:
If this is working in `cidev` is there a reason we're removing it from the taskdef?

Reply:
it was originally set in the start.sh in Mesos and not in the config file - https://github.com/companieshouse/restricted-word-web/blob/master/start.sh#L15
So we thought it would be better to follow the same pattern and have it set in the docker_start.sh (which replaces the start.sh)